deploy/aws/java11Exec/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-core</artifactId>
-            <version>1.3</version>
+            <version>3.0</version>
```

Comment:
This should be updated to the following given [Hamcrest's instructions](https://hamcrest.org/JavaHamcrest/distributables#upgrading-from-hamcrest-1x):

```xml
    <groupId>org.hamcrest</groupId>
    <artifactId>hamcrest</artifactId>
    <version>3.0</version>
```